src/components/workspace/qr-gen.tsx

```diff
@@ -0,0 +1,30 @@
+import { createEffect, createSignal } from "solid-js";
+import * as qrcode from "qrcode";
+
+export default function CertQrCode({ value }: { value: string }) {
```

Comment:
The compiler will warn you that Solid JS does not allow unpacking parameters in the function's definition.
Use `props: { value: string }` instead.